expression/expression.go

```diff
@@ -290,7 +290,7 @@ func ColumnInfos2ColumnsWithDBName(dbName, tblName model.CIStr, colInfos []*mode
 
 // NewCastFunc creates a new cast function.
 func NewCastFunc(tp *types.FieldType, arg Expression, ctx context.Context) Expression {
-	return FoldConstant(BuildCastFunction(ctx, arg, tp))
+	return BuildCastFunction(ctx, arg, tp)
```

Comment:
What is the difference between `NewCastFunc` and `BuildCastFunction` ??

Reply:
Yes, no difference now.